packages/core/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Very Good Ventures
```

Comment:
I assume this should be `Copyright (c) 2022 Widgetbook GmbH`.

Reply:
I made the change but also moved the LICENSE file to the root folder e851c187903e6ec543f508a9b4b54a6afe88d100. What do you think about this?

Reply:
works for me